CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.16)
 project(halide-sar-app)
 
+find_package(MPI REQUIRED)
```

Comment:
Pedantic: please move this down with other `find_package` uses, e.g., after `find_package(Halide REQUIRED)`